### PR TITLE
Adds documentation about git configuration

### PR DESCRIPTION
**Pull request by AlexArdelean21**
In order to solve the task from the git/Github course at Codecademy, I added a markdown document that briefly details some basic configuration options for the git CLI tool. 

Please don't be too hard on me as this is my first PR, and I would appreciate constructive feedback.

### CODE REVIEW

File: content/git/concepts/config/config.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Git Config
```
Comment (caupolicandiaz) [inline content/git/concepts/config/config.md:2]: This metadata section is incorrect please see look at another similar term entry or a template like the example linked - 
https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md?plain=1

File: content/git/concepts/config/config.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+# Git Config
+
+**Git configuration** is a fundamental aspect of using Git, allowing users to customize various settings and preferences on both a global and repository-specific level. Configuration settings in Git help you manage aspects such as your identity, preferred text editor, merge strategies, and much more. Git stores these settings in configuration files, and understanding how to work with these configurations is essential for efficient and smooth collaboration with Git.
```
Comment (caupolicandiaz) [inline content/git/concepts/config/config.md:7]:
```suggestion
**Git configuration** is a fundamental aspect of using Git, allowing users to customize various settings and preferences on both a global and repository-specific level. Configuration settings help users manage various aspects of the profile and operation of Git including: identity details, preferred text editor, merge strategies, and much more. Git stores these settings in configuration files, and understanding how to work with these configurations is essential for efficient and smooth collaboration with Git.
```

Comment (caupolicandiaz) [reply content/git/concepts/config/config.md:7]: docs avoids the use of first and second person pronouns ( I you we). Please update the rest of the document accordingly.

File: content/git/concepts/config/config.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Viewing Git Configurations
+
+You can view your Git configuration using the `git config` command. To see your global configuration settings, use:
+
+```bash
```
Comment (caupolicandiaz) [inline content/git/concepts/config/config.md:13]:
```suggestion
```shell
```

File: content/git/concepts/config/config.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+## Setting Git Configurations 
+
+You can set your Git configurations using the git config command. For example, to set your name and email address, which are crucial for authorship identification:
+
+```bash
+git config --global user.name "Your Name"
+git config --global user.email "your.email@example.com"
+```
+You can also set repository-specific configurations without the --global flag.
```
Comment (caupolicandiaz) [inline content/git/concepts/config/config.md:25]:
```suggestion
Repository-specific configurations can also be set without the `--global flag`.
```

File: content/git/concepts/config/config.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+## Removing Git Configurations
+
+If you need to remove a specific configuration setting, use the --unset flag with the git config command. For example, to remove the user email from your global configuration:
```
Comment (caupolicandiaz) [inline content/git/concepts/config/config.md:29]:
```suggestion
To remove a specific configuration setting, use the `--unset` flag with the `git config` command. For example, to remove the user email from your global configuration:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@AlexArdelean21 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

File: content/git/concepts/config/config.md (hunk 5 above)
Comment (AlexArdelean21) [reply content/git/concepts/config/config.md]: I've revised the document again. Please review it, and if it's not good enough, feel free to let me know, and I'll make further improvements.
@caupolicandiaz

**Comment by caupolicandiaz** [comment]
@AlexArdelean21 This is still missing the metadata section. I provided a reference link the previous review. Please revise

**Review by caupolicandiaz — APPROVED** [review]
@AlexArdelean21 I made a few additional direct edits to update some of the formatting and spacing. Looks good for a second review

File: content/git/concepts/config/config.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+## Setting Git Configurations
+
+Git configurations can be set using the `git config` command. For instance, to set name and email address, which are crucial for authorship identification:
+
+```shell
+git config --global user.name "Your Name"
+git config --global user.email "your.email@example.com"
+```
+
+Repository-specific configurations can also be set without the `--global flag`.
```
Comment (SSwiniarski) [inline content/git/concepts/config/config.md:34]:
```suggestion
Repository-specific configurations can also be set without the `--global` flag.
```

File: content/git/concepts/config/config.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+## Removing Git Configurations
+
+To remove a specific configuration setting, use the `--unset` flag with the `git config` command. For example, to remove the user email from your global configuration:a
```
Comment (SSwiniarski) [inline content/git/concepts/config/config.md:38]:
```suggestion
To remove a specific configuration setting, use the `--unset` flag with the `git config` command. For example, to remove the user email from the global configuration:
```

File: content/git/concepts/config/config.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Git Config'
```
Comment (SSwiniarski) [inline content/git/concepts/config/config.md:2]:
```suggestion
Title: 'Config'
```

File: content/git/concepts/config/config.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+Description: 'Learn how to configure Git settings for smooth collaboration.'
```
Comment (SSwiniarski) [inline content/git/concepts/config/config.md:3]:
```suggestion
Description: 'Git configuration is a fundamental aspect of using Git, allowing users to customize various settings and preferences on both a global and repository-specific level.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@AlexArdelean21 I left a few comments. Also you need to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=3672).

**Comment by AlexArdelean21** [comment]
I have made the changes. @SSwiniarski

**Review by SSwiniarski — APPROVED** [review]
@AlexArdelean21 Looks good.

**Comment by AlexArdelean21** [comment]
Is it possible to link my username to the contributors? I want to show it to my parents. @SSwiniarski

**Comment by SSwiniarski** [comment]
> Is it possible to link my username to the contributors? I want to show it to my parents. @SSwiniarski

@AlexArdelean21 look [here](https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036).

**Comment by AlexArdelean21** [comment]
@SSwiniarski I did that. Thank you.

**Pull request merged**